integration/currentop_test.go

```diff
@@ -32,8 +32,11 @@ import (
 	"github.com/FerretDB/FerretDB/v2/integration/setup"
 )
 
-func TestCurrentOpGetMore(t *testing.T) {
-	t.Parallel()
+func TestCurrentOpGetMore(tt *testing.T) {
+	tt.Parallel()
+
+	// TODO https://github.com/FerretDB/FerretDB/issues/4794
+	t := setup.FailsForFerretDB(tt, "https://github.com/FerretDB/FerretDB/issues/4794")
```

Comment:
I can't tell why other tests (not this test) are failing on CI. It looks like something we've seen before, context getting cancelled.

Reply:
The cause was due to `setup.SetupWithOpts(t, ...)` instead of `setup.SetupWithOpts(tt, ...)`